nuget/ExcelDnaDoc-Template.dna

```diff
@@ -1,21 +1,24 @@
-<DnaLibrary Name="%ProjectName% Add-In" RuntimeVersion="v4.0">
-  <ExternalLibrary Path="%OutputFileName%" LoadFromBytes="true" Pack="true" ExplicitExports="true" />
+<?xml version="1.0" encoding="utf-8"?>
+<DnaLibrary Name="%ProjectName% Add-In" RuntimeVersion="v4.0" xmlns="http://schemas.excel-dna.net/addin/2018/05/dnalibrary">
+  <ExternalLibrary Path="%OutputFileName%" ExplicitExports="false" LoadFromBytes="true" Pack="true" IncludePdb="false" />
   <Reference Path="ExcelDna.Documentation.dll" Pack="true" />
-  
-  <!-- 
+
+  <!--
        The RuntimeVersion attribute above allows two settings:
-       * RuntimeVersion="v2.0" - for .NET 2.0, 3.0 and 3.5
        * RuntimeVersion="v4.0" - for .NET 4 and 4.5
+       * RuntimeVersion="v2.0" - for .NET 2.0, 3.0 and 3.5
```

Comment:
I think these lines are now redundant?

Reply:
I don't think so (i.e. not yet)... We still need to release a final version of ExcelDnaDoc which targets Excel-DNA v1.1

Reply:
Sorry - you're right. I'm living in the future.